### PR TITLE
No need to check for existence and consistence of quality_control_set

### PR DESCRIPTION
**Pull request by ggalibert**

### CODE REVIEW

File: compliance_checker/imos/imos.py [hunk 1]
```diff
@@ -1255,8 +1255,8 @@ def check_data_variables(self, dataset):
 
     def check_quality_control_conventions_for_quality_control_variable(self, dataset):
         """
-        Check that the attributes quality_control_set and quality_control_conventions
-        are valid and consistent.
+        Check that the attribute quality_control_conventions
+        is valid and consistent.
         """
         test_value_dict = {'1': "IMOS standard set using the IODE flags",\
```
Comment (mhidas) [inline compliance_checker/imos/imos.py]: This doesn't need to be a dict now. A tuple would be ideal (e.g. `qc_conventions = ("IMOS standard set using the IODE flags", etc...)`

**Comment by mhidas** [comment]
@ggalibert Looks good, thanks. Just one small thing to fix (see my in-line comment)

**Comment by ggalibert** [comment]
Ok, just pushed force the required changes in your last comment.

File: compliance_checker/imos/imos.py [hunk 2]
```diff
@@ -1255,35 +1255,16 @@ def check_data_variables(self, dataset):
...
-        are valid and consistent.
+        Check that the attribute quality_control_conventions
+        is valid and consistent.
         """
-        test_value_dict = {'1': "IMOS standard set using the IODE flags",\
-                           '2': "ARGO quality control procedure",\
-                           '3': "BOM (SST and Air-Sea flux) quality control procedure",\
-                           '4': "WOCE quality control procedure"}
+        test_value_conventions = ("IMOS standard set using the IODE flags",\
+                           "ARGO quality control procedure",\
```
Comment (mhidas) [inline compliance_checker/imos/imos.py]: To make this easier to read, could you indent so that all items in the tuple start in the same column?

**Comment by ggalibert** [comment]
With tuple now nicely indented.

**Comment by mhidas** [comment]
:+1: Thanks @ggalibert !

**Pull request merged**